e2e/testconfig/testconfig.go

```diff
@@ -197,6 +199,16 @@ func defaultModifyGenesis(denom string) func([]byte) ([]byte, error) {
 
 		appState[govtypes.ModuleName] = govGenBz
 
+		icaGenesisState := icagenesistypes.DefaultGenesis()
+		icaGenesisState.HostGenesisState.Params.AllowMessages = []string{"*"}
+
+		icaGenBz, err := cdc.MarshalJSON(icaGenesisState)
+		if err != nil {
+			return nil, fmt.Errorf("failed to marshal interchain accounts genesis state: %w", err)
+		}
+
+		appState[icatypes.ModuleName] = icaGenBz
```

Comment:
I guess this is not required anymore. ref: #2290

Reply:
Yea lets just remove and test this functionality against >= v6